File: crates/axhub-helpers/tests/fixtures/ast-validate/go/good.go
package sample

// owner-scoped 테이블은 무필터 list/count 가 정당해요.
func loadPosts(client *Client, ownerID string) {
	mine := client.table("posts").list()
	total := client.table("posts").count()
	filtered := client.table("posts").eq("owner_id", ownerID).limit(20).list()
	_ = mine
	_ = total
	_ = filtered
}

File: crates/axhub-helpers/tests/fixtures/ast-validate/java/good.java
package sample;

// owner-scoped 테이블은 무필터 list/count 가 정당해요.
class Sample {
    void loadPosts(Client client, String ownerId) {
        var mine = client.table("posts").list();
        var total = client.table("posts").count();
        var filtered = client.table("posts").eq("owner_id", ownerId).limit(20).list();
        System.out.println(mine + total + filtered);
    }
}
